docs/en_US/CommunitySharings/NNI_colab_support.md

```diff
@@ -0,0 +1,44 @@
+
+# Use NNI on Google Colab
+NNI can easily run on Google Colab platform. However, colab doesn't expose its public IP and ports, so by default you can not access NNI's Web UI on colab. To solve this, you need a reverse proxy software like `ngrok` or `frp`. This tutorial will show you how to use ngrok to access NNI's Web UI on colab.

```

Comment:
There is still a lot of "colab". Please change them to "Colab".

Reply:
My bad. Fixed.